src/scss/global/_archmage-global.scss

```diff
@@ -601,6 +601,17 @@ option[selected] {
         }
       }
     }
+
+    @import '../v2/utils/typography';
+    @import '../v2/utils/variables';
+    @import '../v2/utils/colors';
+    @import '../v2/components/character/effects';
```

Comment:
This is a bit of a hacky way to backport the v2 effect styles into the v1 item sheets, but it's good enough for now IMO since we're still planning to eventually retire these sheets.